templates/metastore.yml

```diff
@@ -0,0 +1,102 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Metastore Stack
```

Comment:
Per IRC conversation, the metastore uses the worker nodes when running "MSCK REPAIR TABLE".